src/cascade/model/session.py

```diff
@@ -349,3 +345,62 @@ def simulation(self, index):
             the Nth simulation.
         """
         return self._file_objects.read_simulation_model_and_data(self._model, self._data, index)
+
```

Comment:
These functions are of a piece, and they might make sense in their own file?

Reply:
Bleh. I'm struggling with which package that file would be in. It clearly doesn't belong in model. It seems like it belongs in executor but I think you said you didn't want other packages to have dependencies on executor, which is totally reasonable. Ok I'm going to make `cascade.core.subprocess_utils` That's not perfect but I think it's good enough and if we want to move it later, that's easy.